open-sauced/intro
# fix: Update intro.md

Eliminate extraneous '>'  from 2nd paragraph of "Prerequisites". (replace "starting> this course" with "starting this course").

## Description

This PR removes an extraneous '>', an apparent typo, from the 2nd paragraph of "Prerequisites"


## Related Tickets & Documents

Fixes #192 


## Mobile & Desktop Screenshots/Recordings
_**Original (branch main):**_

<img width="836" alt="p2" src="https://github.com/open-sauced/intro/assets/71496139/4ddac38d-faab-4532-a3ff-e45035c4c69f">

_**PR will fix to display:**_


<img width="821" alt="p1" src="https://github.com/open-sauced/intro/assets/71496139/d5cd32ec-c08c-4425-9168-44ad5de1e6b8">




## Steps to QA

QA process:
- Navigate to [https://github.com/georgepence/intro/blob/patch-1/docs/intro-to-oss/intro.md](https://github.com/georgepence/intro/blob/patch-1/docs/intro-to-oss/intro.md)
  - Note that the third paragraph ends with: "... learn the basics before starting this course."
  - Note that the [original version](https://github.com/open-sauced/intro/blob/main/docs/intro-to-oss/intro.md) paragraph three ends with "... learn the basics before starting**>** this course." (bold mine)




## Tier (staff will fill in)


## [optional] What gif best describes this PR or how it makes you feel?
